src/views/listview/list-view.html

```diff
@@ -1,52 +1,64 @@
-<div class="list-group list-view-pf">
+<div class="list-group list-view-pf" dnd-list="items" ng-class="{'list-view-pf-dnd': config.dragEnabled === true}">
+  <div class='dndPlaceholder'></div>
```

Comment:
The placeholder div has to be there; otherwise, I get an ugly empty space at the top of the list when enabled.

![screen shot 2017-01-27 at 10 04 29 am](https://cloud.githubusercontent.com/assets/17481322/22375469/2de4c7d6-e478-11e6-9b67-1ead8063860b.png)